ups/ap_pipe.cfg

```diff
@@ -0,0 +1,26 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+# Dependencies that provide header files and or libraries should be included here.
+# Pure-Python dependencies do not need to be included.
+# Packages that use pybind11 or boost_tests should declare them as build dependencies.
+# Otherwise, the rules for which packages to list here are the same as those for
+# table files.
+dependencies = {
+    "required": ["utils", "ndarray"],
+    "buildRequired": ["boost_test", "pybind11", "ndarray", "pex_exceptions", "obs_decam"],
+    "optional": ["ap_verify_hits2015", "ap_verify"],
```

Comment:
I don't think `ap_pipe.cfg` needs to contain many packages, particularly not Python-only ones, because it's used by Scons to set up paths and the like. But I'll admit I don't know for sure.